src/store.c

```diff
@@ -437,6 +439,23 @@ static int _must_use_ _nonnull_ cs_content_add(struct clip_store *cs,
         }
 
         return 0;
+    } else if (dupe && deduplicate) {
```

Comment:
`cs_content_add` is now getting pretty unwieldy. Please split into two commits:

1. Split the existing `dupe` logic in `cs_content_add` `_cs_content_add_link`
2. Add your own logic as `_cs_content_add_move_last`

Make sure to add the docs per the rest of the file. Thanks!